group_vars/all.yml

```
@@ -1647,6 +1678,8 @@ varnish_strip_params:
   - "{{ https_termination_redirect_source_domain_param }}"
 
 varnish_debug_request_info_header_name: "{{ mageops_debug_http_header_prefix }}-Info-Varnish"
+varnish_qa_request_info_header_name: "{{ mageops_qa_http_header_prefix }}-Varnish"
```

Maybe we should also use `-Info-Varnish` suffix to keep things consistent?


```suggestion
varnish_qa_request_info_header_name: "{{ mageops_qa_http_header_prefix }}-Info-Varnish"
```